packages/muelu/src/CMakeLists.txt

```diff
@@ -353,7 +353,9 @@ LIST(REMOVE_ITEM HEADERS ${HEADERS_INTERFACE})
 ###############################################
 SET(SOURCES ${MUELU_ETI_CPP_SOURCES_SLGN})
 APPEND_GLOB(SOURCES */*.cpp)
+LIST(REMOVE_ITEM SOURCES ${DIR}/Graph/MueLu_LinkedList.cpp)
```

Comment:
Does a file really just need to be moved in a subdir or something?  Is this going to confuse people?